openslides_backend/action/actions/poll/create.py

```diff
@@ -0,0 +1,150 @@
+from typing import Any, Dict
+
+from ....models.models import Poll
+from ....shared.patterns import FullQualifiedId
+from ....shared.schema import optional_fqid_schema
+from ...generics.create import CreateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ..option.create import OptionCreateAction
+
+options_schema = {
+    "description": "A option inside a poll create schema",
+    "type": "object",
+    "properties": {
+        "text": {"type": "string", "description": "the text of an option"},
+        "content_object_id": optional_fqid_schema,
+        "Y": {"type": "string", "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$"},
```

Comment:
Please extract this regex as a constant and import it from somewhere it makes most sense

Reply:
Extracted the decimal schema and moved it into the schema.py file.